package.json

```diff
@@ -22,7 +22,7 @@
   },
   "homepage": "https://github.com/npm/policies",
   "dependencies": {
-    "marked": "^0.3.5",
-    "mkp": "^1.0.1"
+    "@npm/mkp": "^1.0.2",
```

Comment:
Good catch - I recall there were some instances where we were able to get off of private [outdated] forks and rely back on mainline packages, but I'm not sure what the context for this one is.